app/assets/stylesheets/components/_subheader.scss

```diff
@@ -3,8 +3,13 @@
 }
 
 .global-subheader {
-  padding-top: $container-v-padding;
+  padding-top: 12px;
+  padding-bottom: 12px;
```

Comment:
No. Because this element uses horizontal padding from higher in the cascade. We only want to set the vertical padding at this point.